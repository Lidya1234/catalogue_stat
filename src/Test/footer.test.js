import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../components/footer';
import '@testing-library/jest-dom/extend-expect';

describe('The Footer should match with snapshot', () => {
  const { getByText } = render(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>,
  );
  test('Matchs snapshot', () => {
    expect(getByText).toMatchSnapshot();
  });
  
  const component = renderer.create(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>,
  ).toJSON();

  test('Matchs snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
