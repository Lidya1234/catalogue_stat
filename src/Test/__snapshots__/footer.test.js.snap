// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer should match with snapshot Matchs snapshot 1`] = `[Function]`;

exports[`The Footer should match with snapshot Matchs snapshot 2`] = `
<footer
  className="footer"
>
  <p
    className="footer-txt"
  >
     All Rights Reserved Lidya.G @2021
  </p>
</footer>
`;

exports[`The navigation bar should contain Matchs snapshot 1`] = `[Function]`;
